main.go

```diff
@@ -33,12 +34,23 @@ const listenAddress = ":8210"
 
 func main() {
 	logrus.Info("Starting the Hashicorp Vault Proxy...")
+
 	pathPrefix := os.Getenv("VAULT_PATHPREFIX")
 	if pathPrefix == "" {
 		pathPrefix = "kv"
 	}
 
-	kv, err := vault.NewKVStore(pathPrefix)
+	pathName := os.Getenv("VAULT_PATHNAME")
+	if pathName == "" {
```

Comment:
it can't be empty? There's a Getenv variant that returns a bool whether it's set